core/ui/src/views/settings/SettingsSystemLogs.vue

```diff
@@ -54,116 +54,125 @@
       :showCloseButton="false"
     />
     <cv-row v-if="loading.lokiInstances && loading.subscription">
-      <cv-column :md="4" :xlg="4">
-        <NsTile :light="true" :icon="Catalog32">
-          <cv-skeleton-text
-            :paragraph="false"
-            :line-count="1"
-            heading
-          ></cv-skeleton-text>
-          <cv-skeleton-text
-            :paragraph="true"
-            :line-count="9"
-          ></cv-skeleton-text>
-        </NsTile>
+      <cv-column>
+        <!-- skeleton card grid -->
+        <div
+          class="card-grid grid-cols-1 sm:grid-cols-2 xl:grid-cols-3 3xl:grid-cols-4"
```

Comment:
```suggestion
          class="card-grid grid-cols-1 md:grid-cols-2 2xl:grid-cols-3 3xl:grid-cols-4"
```
Let's use for skeleton the same card sizes as the actual card grid